references/rpm_ostree_command_ref.adoc

```diff
@@ -0,0 +1,251 @@
+  glibc-common 2.25-8.fc26 -> 2.25-7.fc26
+  krb5-libs 1.15.1-22.fc26 -> 1.15.1-21.fc26
+  libcrypt-nss 2.25-8.fc26 -> 2.25-7.fc26
+  python2-setuptools 36.2.0-2.fc26 -> 36.2.0-1.fc26
+  python3 3.6.2-5.fc26 -> 3.6.2-1.fc26
+  python3-libs 3.6.2-5.fc26 -> 3.6.2-1.fc26
+  python3-setuptools 36.2.0-2.fc26 -> 36.2.0-1.fc26
+  system-python 3.6.2-5.fc26 -> 3.6.2-1.fc26
+  system-python-libs 3.6.2-5.fc26 -> 3.6.2-1.fc26
+  tmux 2.5-4.fc26 -> 2.5-1.fc26
+Run "systemctl reboot" to start a reboot
+....
+
+== Hybrid image/packaging via package layering
+
+It is possible to dynamically add more packages onto the system that are not
+part of the commit composed on the server. These additional "layered" packages
+are persistent across upgrades, rebases, and deploys (contrast with the
+link:https://github.com/ostreedev/ostree/blob/master/man/ostree-admin-unlock.xml[ostree unlocking]
+mechanism).
+
+This is where the true hybrid image/package nature of `rpm-ostree` comes into
+play; you get a combination of the benefits of images and packages.  The
+package updates are still fully transactional and offline.
+
+For example, you can use package layering to install 3rd party
+kernel modules, or userspace driver daemons such as `pcsc-lite-ccid`.
+While most software should go into a container, you have full flexibilty
+to use packages where it suits.
+
+
+=== `rpm-ostree install/uninstall <pkg>`
+
+The `rpm-ostree install` command will download the target package, its dependencies,
+and create a new deployment with those packages installed.  Additionally, you can use
+the `rpm-ostree install` to layer local RPMs onto the system.  To remove layered
+packages, use `rpm-ostree uninstall`.
+
+By default, every `rpm-ostree` operation is "offline" - it has no effect
+on your running system, and will only take effect when you reboot.  This
+"pending" state is called the "pending deployment".  Operations can be chained;
+for example, if you invoke `rpm-ostree upgrade` after installing a package, your
+new root will upgraded with the package also installed.
+
+
+== Rebasing via `rpm-ostree rebase`
+
+Your operating system vendor may provide multiple base branches.  For example,
+Fedora Atomic Host has branches of the form:
+
+ - `fedora/26/x86_64/atomic-host`
+ - `fedora/26/x86_64/updates/atomic-host`
+ - `fedora/27/x86_64/atomic-host`
+
+You can use the `rebase` command to switch between these; this can represent a
+major version upgrade, or logically switching between different "testing"
+streams within the same release. Like every other `rpm-ostree` operation, All
+layered packages and local state will be carried across.
+
+The `rebase` command also allows you to switch your system to an entirely new
+OS distribution.  This means it is possible to switch your Fedora 26 Atomic Host
+to a version of CentOS Atomic Host.  While this is not likely to be done on a
+production system, it can be useful in certain testing scenrios.
+
```

Comment:
We should add a relative link here to the package layering doc. Maybe something like:

```
See the link:../using_ostree/ostree_mutations/package_layering.adoc[package layering section] for more information on package layering. 
```

?

Reply:
Good idea